platform_plugin_forum_email_notifier/templates/platform_plugin_forum_email_notifier/edx_ace/digestemailnotification/email/body.html

```diff
@@ -2,32 +2,42 @@
 
 {% load i18n %}
 {% load static %}
+
 {% block content %}
 <table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
     <tr>
         <td>
             <h1>
-                {% trans "Forum Digest" as tmsg %}{{ tmsg | force_escape }}
+                {% trans "Summary of Discussion Activity" as tmsg %}{{ tmsg | force_escape }}
             </h1>
-            <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
-                {% blocktrans %}Activity in the discussion:{% endblocktrans %}
-                {% endfilter %}
-                <br />
-            </p>
 
-            {% filter force_escape %}
-                {% blocktrans asvar course_cta_text %}Visit discussions{% endblocktrans %}
-            {% endfilter %}
-            {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=confirm_link %}
+            {% for thread in threads %}
+                {% if thread.object_type == 1 %}
+                    <p style="color: rgba(0,0,0,.75);">
+                        <a href="{{ thread.url }}discussions/{{ course_id }}/posts/{{ thread.thread_id }}">{{ thread.title }} ↗</a>: {{ thread.body }}
+                        <br />
+                    </p>
+                {% elif thread.object_type == 2 %}
+                    <p style="color: rgba(0,0,0,.75);">
```

Comment:
Same as [here](https://github.com/eduNEXT/platform-plugin-forum-email-notifier/pull/5/files#r1365598524) for the other paragraphs

Reply:
could we use a style block on top of the file?

Reply:
As @mariajgrimaldi comments: "Considering that in edx-platform the email styles are like this, we can leave them like this. In other implementations where we know it works well in another way, we change it"